wagtail/contrib/table_block/blocks.py

```diff
@@ -10,6 +10,7 @@
 
 
 DEFAULT_TABLE_OPTIONS = {
+    'table_header_choice': 'row',
```

Comment:
At first glance I think it would be better for there to be no default so editors are forced to choice.

Reply:
I removed having a default for all TableBlocks but will probably add back a way for developers to set a default when defining their own table blocks